torch/csrc/distributed/c10d/CUDASymmetricMemory.cu

```diff
@@ -1,275 +1,32 @@
-#include <torch/csrc/distributed/c10d/CUDASymmetricMemory.hpp>
 #include <torch/csrc/distributed/c10d/CUDASymmetricMemory-inl.h>
+#include <torch/csrc/distributed/c10d/CUDASymmetricMemory.hpp>
+#include <torch/csrc/distributed/c10d/CUDASymmetricMemoryUtils.hpp>
 #include <torch/csrc/distributed/c10d/cuda/utils.hpp>
 
 #include <ATen/ceil_div.h>
 #include <ATen/cuda/CUDAContext.h>
 #include <c10/cuda/CUDACachingAllocator.h>
 #include <c10/cuda/CUDAGuard.h>
-#include <c10/util/env.h>
 #include <c10/util/error.h>
 
+#include <sys/socket.h>
+#include <unistd.h>
+
 #if !defined(USE_ROCM) && defined(PYTORCH_C10_DRIVER_API_SUPPORTED)
 #include <c10/cuda/driver_api.h>
 #endif
 
-#include <sys/socket.h>
-#include <sys/syscall.h>
-#include <sys/un.h>
-#include <unistd.h>
-
 #if defined(CUDART_VERSION) && CUDART_VERSION >= 12030
 #define CUDART_SUPPORTS_MULTICAST
 #endif
 
-namespace {
```

Comment:
you should really do the code-motion part in a previous PR so its easier to tell what is real changes